packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -61,6 +63,10 @@ export default class PouchLink extends CozyLink {
     this.synced = window.localStorage.getItem(LOCALSTORAGE_SYNCED_KEY) || false
   }
 
+  getPouchOptions() {
+    return pickBy(this.options, (value, key) => startsWith(key, 'pouch'))
```

Comment:
Instead of prefixing with `pouch`, wouldn't be simpler to do:

```
{
  pouch: {
    options: {},
    plugins: [] 
  }
}
```

Reply:
I change it 👍